lib/paperclip/storage/eitheror.rb

```diff
@@ -73,7 +73,7 @@ def usable_storage
         either_exists = @either.exists?
         or_exists = @or.exists?
 
-        return @either if either_exists || !or_exists
+        return @either if !or_exists || either_exists
 
         options[:autosync] && sync ? @either : @or
```

Comment:
I guess if we get to the point of attempting an `autosync` then it's because `!or_exists` and `either_exists` both returned false, then, the asset **is present** in the NFS, thus, a sync operation is viable.